kubernetes_asyncio/.swagger-codegen/COMMIT

```diff
@@ -1,2 +0,0 @@
-Requested Commit: f9b2839a3076f26db1b8fc61655a26662f2552ee
```

Comment:
I had to delete `kubernetes_asyncio/.swagger-codegen` and rename `kubernetes_asyncio/.swagger-codegen-ignore` to `kubernetes_asyncio/.openapi-generator-ignore`.